### PR TITLE
Views tests

### PR DESCRIPTION
**Pull request by yusufsafak**
#953 rebased and fixed conflicts

### CODE REVIEW

File: packages/node_modules/cerebral/src/views/preact/StateContainer.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ class StateContainer extends Component {
     }
   }
   render() {
-    return this.props.children
+    return this.props.children[0]
```
Comment (yusufsafak) [inline packages/node_modules/cerebral/src/views/preact/StateContainer.js]: Btw. Preact Containers don't support more than one child :(
it renders only first children.  you can't use it like 
```
<StateContainer>
    <TestComponent />
    <AnotherComponent />
</StateContainer>
```
It does not render AnotherComponent.

Comment (christianalfoni) [inline packages/node_modules/cerebral/src/views/preact/StateContainer.js]: Does this mean it is fixed? :) `StateContainer` is legacy stuff btw, `UniversalController` is really the replacement

Comment (christianalfoni) [inline packages/node_modules/cerebral/src/views/preact/StateContainer.js]: Though nice for testing, but yeah... totally fine this change :)

**Comment by Guria** [comment]
Sorry @yusufsafak Seems needs once more rebase now :(

**Pull request closed without merging**